src/components/binding/binding.js

```diff
@@ -57,8 +57,10 @@ genotet.bindingData;
 
 /**
  * @typedef {{
- *   fileName: string,
- *   chr: string
+ *   fileNames: (!Array<string>|string),
+ *   bedName: string,
+ *   chr: string,
+ *   isPreset: boolean
```

Comment:
What's the purpose of creating isPreset variable? I think view is supposed not to know whether itself is preset. What you are differentiating here is whether the view is loading multiple track data. So "multipleTracks" is what I suggested.